src/index.html

```diff
@@ -1,13 +1,686 @@
+                </p>
+                <a
+                  href="https://goo.gl/maps/z9TYGjLrDJ82hjEb7"
+                  target="_blank"
+                  class="contact-us__contact link"
+                >
+                  1000 Fifth Avenue
+                  <br>
+                  New York, NY 10028
+                </a>
+              </div>
+            </div>
+
+          </div>
+
+          <div
+            class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-4-6"
+          >
+            <div class="contact-us__contacts-container">
+              <h3 class="contact-us__museum-name">
+                The Met Cloisters
+              </h3>
+              <div class="contact-us__contact-container">
+                <p class="contact-us__contact-type">
+                  Phone
+                </p>
+                <a
+                  href="tel:+1 212-923-3700"
+                  class="contact-us__contact link"
+                >
+                  +1 212-923-3700
+                </a>
+              </div>
+
+              <div class="contact-us__contact-container ">
+                <p class="contact-us__contact-type">
+                  Adress
+                </p>
+                <a
+                  href="https://goo.gl/maps/dFEyAt9C3HJku6Yu9"
+                  target="_blank"
+                  class="contact-us__contact link"
+                >
+                  99 Margaret Corbin Drive
+                  <br>
+                  Fort Tryon Park
+                  <br>
+                  New York, NY 10040
+                </a>
+              </div>
+            </div>
+
+          </div>
+
+          <div
+            class="
+              grid__item
+              grid__item--tablet-4-6
+              grid__item--desktop-8-12
+              grid__item--tablet-rows"
+          >
+            <form
+              action="#"
+              method="get"
+              id="form"
+            >
+              <input
+                name="name"
+                type="text"
+                class="form-field contact-us__field"
+                placeholder="Name"
+                required
+              >
+              <input
+                name="email"
+                type="email"
+                class="form-field contact-us__field"
+                placeholder="Email"
+                required
+              >
+              <textarea
+                name="message"
+                class="
+                  form-field
+                  form-field--textarea
+                  contact-us__field--last
+                  contact-us__field"
+                placeholder="Message"
+                required
+              ></textarea>
+
+              <button class="button button--form" >
+                Send
+              </button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer class="footer">
+    </footer>
```

Comment:
if it is empty, keep it on the same line
```suggestion
    <footer class="footer"></footer>
```